qubes-dist-upgrade.sh

```diff
@@ -312,8 +328,12 @@ if [ "$assumeyes" == "1" ] || confirm "-> Launch upgrade process?"; then
         # shellcheck disable=SC2086
         qubes-dom0-update ${dnf_opts_noclean} --downloadonly --force-xen-upgrade --action=distro-sync || exit_code=$?
         if [ -z "$exit_code" ] || [ "$exit_code" == 100 ]; then
-            if [ "$assumeyes" == "1" ] || confirm "---> Shutdown all VM?"; then
-                qvm-shutdown --wait --all
+            if [ "$assumeyes" == "1" ] || confirm "---> Shutdown all qubes (requested keep running and input devices qubes will remain powered on)?"; then
+                qvm_shutdown_exclude=()
+                for qube in "${extra_keep_running[@]}"; do
+                    qvm_shutdown_exclude+=("--exclude $qube")
```

Comment:
This doesn't work: https://openqa.qubes-os.org/tests/99213#step/release_upgrade/29
I think you need:
```suggestion
                    qvm_shutdown_exclude+=("--exclude" "$qube")
```
or maybe with `=` ?

Reply:
It works with `=`.